.github/workflows/dockerhub-release.yml

```diff
@@ -91,11 +91,6 @@ jobs:
         env:
           SKOPEO_AUTH_DIR: ${{env.SKOPEO_AUTH_DIR}}
         run: |
-          # limit what we promote to latest for the current year only
-          current_year=$(date +%Y)
-          current_month=""
-          sed -i "s/CURRENT_YEAR/$current_year/g; s/CURRENT_MONTH/$current_month/g" "${GITHUB_WORKSPACE}/.github/promote-images.yml"
```

Comment:
This here wasn't needed, as we only used the keys and not their values in the `$IMAGES` below